WalletWasabi.Tests/AcceptanceTests/HwiKatas.cs

```diff
@@ -69,7 +69,6 @@ public async Task TrezorTKataAsync()
 
 			// Because of the Trezor T 2.3.5 firmware update,
 			// we cannot use any longer the KeyManager.DefaultAccountKeyPath.
-			// Either this or introduce 2 more static readonly field in KeyManager.
```

Comment:
@adamPetho IMO this line should be removed.

Reply:
ACK